### DISCUSSION TITLE
Blazored/Toast in ASP.net Core NET 6

### DISCUSSION THREAD

**Discussion by BigMac0069** [Blazored/Toast#197] [Q&A]
Hello,
I'm trying to use your Blazored/Toast in a ASP.net Core NET 6 project and in have a trouble of css.
In fact, the toast is showing but not at the good place and without css application.
Here is the toast :
BlazoredToasts Position="ToastPosition.BottomRight"
                Timeout="10"
                IconType="IconType.Material"
                ErrorIcon="error_outline"
                InfoIcon="school"
                SuccessIcon="done_outline"
                WarningIcon="warning"
                ShowProgressBar="@true"
                ShowCloseButton="@true"
                MaxToastCount="3">
    <CloseButtonContent
        <div>
            <span class="myCloseButtonStyleClass">&times;</span
        </div>
    </CloseButtonContent>
</BlazoredToasts
Here is the result :
![image](https://user-images.githubusercontent.com/117302111/217502654-ddc199ce-010e-4908-89cc-ea99bb7da3b8.png)

I use a component razor that i created and i don't have a _host.cshtml, only a _layout.cshtml.
Can you give me some assistance, please.

Regards,

**Comment by chrissainty** [comment]
Are you using scoped CSS? If not you'll need to add a link to the compiled scoped CSS in your host page. It's covered in the readme.

**Reply by BigMac0069** [reply]
I had these in my _Layout.cshtml : 
link href="Election_Manager.styles.css" rel="stylesheet" /
    link href="BlazorServer.styles.css" rel="stylesheet" /
    link href="_content/Blazored.Toast/blazored-toast.css" rel="stylesheet" /
	link rel="stylesheet" href="~/css/site.css" asp-append-version="true" /    

But, when i saw the dev console i saw this : 
![image](https://user-images.githubusercontent.com/117302111/217541343-b74982aa-2250-4b1f-b411-445e26ff61a8.png)
The css seems to be empty :

**Reply by chrissainty** [reply]
It looks like you still have a reference to blazored-toasts.css which doesn't exist in the latest version. Are you using the latest version?

**Reply by BigMac0069** [reply]
I'm using your Toast-main project.
Is it better to use the nugget version ?

**Reply by BigMac0069** [reply]
I remove the blazored-toasts.css and i have the same trouble.
BlazorServer.styles.css seem to be empty.

**Reply by chrissainty** [reply]
You're not using the NuGet package? Have you just cloned the repo and using the source project? If so please use the NuGet package.

**Reply by BigMac0069** [reply]
I just did it and i still got the same trouble.
The BlazorServer.styles.css is empty.
![image](https://user-images.githubusercontent.com/117302111/217562491-442f9f2a-77e2-4e0d-94ce-e2396b0f6001.png)

**Reply by chrissainty** [reply]
Can you create a repor with your project in it so I can take a look. I don't think I can suggest anything else.
